rasa/server.py

```diff
@@ -665,12 +665,17 @@ def create_app(
             domain_path = os.path.join(temp_dir, "domain.yml")
             dump_obj_as_str_to_file(domain_path, rjs["domain"])
 
+        if rjs.get("save_to_default_model_directory", True):
```

Comment:
I think a string comparison might be more robust here, because this will eval to `True` even if it contains some random string. how about 
```suggestion
        if str(rjs.get("save_to_default_model_directory", True)).lower() == "true":
```

Reply:
Good point. But I think a `is True` should be enough, shouldn't it?

Reply:
👍